modules/geo/src/main/java/org/opensearch/geo/search/aggregations/bucket/geogrid/InternalGeoHashGrid.java

```diff
@@ -45,9 +45,9 @@
  *
  * @opensearch.internal
  */
-public class InternalGeoHashGrid extends InternalGeoGrid<InternalGeoHashGridBucket> {
+public class InternalGeoHashGrid extends BaseGeoGrid<InternalGeoHashGridBucket> {
```

Comment:
Haven't looked into code usage, but I believe we could make all `InternalXxx` classes package-private, wdyt?

Reply:
Sure. I can make that as part of another PR. Will that work?

Reply:
But why? I think the change is perfectly scoped into this pull request

Reply:
Got it. I updated Internal**GridBucket to package private. InternalGeo*Grid is used by GeoModulePlugin for Aggregation registration. Hence, i left those two classes as it is.

Reply:
@reta Updated accordingly. Can you please take a look one more time?